cluster/gce/manifests/cluster-autoscaler.manifest

```diff
@@ -17,7 +17,7 @@
         "containers": [
             {
                 "name": "cluster-autoscaler",
-                "image": "k8s.gcr.io/cluster-autoscaler:v1.17.0",
+                "image": "us.gcr.io/k8s-artifacts-prod/autoscaling/cluster-autoscaler:v1.17.2",
```

Comment:
The endpoint here needs to be:

```
k8s.gcr.io/autoscaling/cluster-autoscaler
```

Given this PR was opened in March, the version of cluster-autoscaler has probably changed as well.